Daniel-Mackay/Daniel-Mackay.md

```diff
@@ -0,0 +1,54 @@
+---
+id: "9bf364e8-342f-ed11-9db1-00224818a84c"
+
+qualifications: "Bachelor of Science (Honours), Computer Science"
+
+quote: "The healthy, wealthy, and wise prioritise education over entertainment"
+quoteAuthor: "Jay Shetty"
+---
+
+[[imgBadge]]
+| ![Azure Fundamentals](../badges/Certification-microsoft-azure-fundamentals.png)
+
+[[imgBadge]]
+| ![Azure](../badges/Business-microsoft-azure.png)
+
+[[imgBadge]]
+| ![Azure Functions](../badges/Developer-azure-function.png)
+
+[[imgBadge]]
+| ![.NET Core](../badges/Developer-dotnet-core.png)
+
+[[imgBadge]]
+| ![C#](../badges/Developer-c-sharp.png)
+
+[[imgBadge]]
+| ![JS](../badges/Developer-js.png)
+
+[[imgBadge]]
+| ![React](../badges/Developer-react.png)
+
+[[imgBadge]]
+| ![Azure DevOps](../badges/Business-microsoft-azure-devops.png)
+
+[[imgBadge]]
+| ![DevOps](../badges/Developer-devops.png)
+
+[[imgBadge]]
+| ![PowerShell](../badges/Developer-powershell.png)
+
+Daniel Mackay is a Solution Architect at SSW, and specialises in Cloud Architecture, Software Architecture, and Dev Ops.  He has over 20+ years of professional experience.  He has worked with .NET since 2009, and is passionate about web-based technologies such as .NET Core, ASP.NET Core, EF Core, React, Next.js and Azure.
+
+You can find his blog at [www.dandoescode.com](https://www.dandoescode.com/).
+
+### Cloud Architecture
+
+Daniel loves working with Azure and is constantly trying to identify new solutions to existing problems.  He is experienced with App Service, Function Apps, Service Bus, Storage, Logic Apps, Azure AD, Application Insights and Static Web Apps.
+
+### Software Architecture
+
+Daniel is a hands-on Solution Architect with an interest in both Software Architecture and Cloud Architecture.  He is a big fan of Clean Architecture with .NET Core and the productivity and maintainability it provides. He also has experience building single page applications in React and Angular.
+
+### Dev Ops
```

Comment:
DevOps or Dev Ops 🤔
I'm not sure what is correct, but it should be consistent

Reply:
Good catch @bradystroud.  After a bit of research 'DevOps' seems to be the standard term.